src/main/kotlin/com/katanox/tabour/integration/sqs/core/consumer/SqsEventHandler.kt

```diff
@@ -44,4 +46,8 @@ class SqsEventHandler(
             }
         }
     }
+
+    fun onFailure(throwable: Throwable, message: Any) {
+        failureAction
```

Comment:
just to be sure: you don't need to invoke it as a function?

Reply:
figure it that this is not needed